app/components/rdf-input-fields/mandataris-fractie-selector.js

```diff
@@ -68,42 +69,37 @@ export default class MandatarisFractieSelector extends InputFieldComponent {
   }
 
   async loadProvidedValue() {
-    const membershipTriples = triplesForPath(this.storeOptions, false).values;
-    if (!membershipTriples.length) {
+    const fractieUris = triplesForPath(this.storeOptions, false).values;
+    if (!fractieUris.length) {
       return;
     }
-    const membershipUri = membershipTriples[0].value;
+    const fractieUri = fractieUris[0].value;
 
-    const matches = await this.store.query('lidmaatschap', {
-      'filter[:uri:]': membershipUri,
-      include: 'binnen-fractie',
+    const matches = await this.store.query('fractie', {
+      'filter[:uri:]': fractieUri,
     });
-    this.membership = matches.at(0);
+    this.selectedFractie = matches.at(0);
   }
 
+  // TODO Maybe turn into task?
   @action
   async onSelectFractie(fractie) {
```

Comment:
Maybe should be a task? You get `this.updating = true` at the start and `this.updating = false` at the end for free with tasks

Reply:
this is true. If you feel it's worth the effort you can ofc do this